weld/sir.rs

```diff
@@ -49,6 +49,11 @@ pub enum Statement {
         index: Symbol,
         size: Symbol,
     },
+    Sort {
+        output: Symbol,
+        child: Symbol,
+        keyfunc: SirFunction,
```

Comment:
Thoughts on this approach?
I didn't find a way to make a Lambda symbol and using it properly.
There probably is a solution where having a Lambda symbol here is possible.

Reply:
I think this is fine, I think the function will get inlined so there shouldn't be  performance issue ... you should run performance numbers though to confirm